File: bootstrap/v0.0.8.4.5/benchmarks/python/hashtable.py
def main():
    table = {}

    # Insert 10000 key-value pairs
    for i in range(10000):
        table[i] = i * 2

    # Lookup 10000 values
    total = 0
    for i in range(10000):
        if i in table:
            value = table[i]
            total += value

    return 0

if __name__ == "__main__":
    exit(main())

File: bootstrap/archived_runa_0.0.7.5+/v0.0.8.4.5.1/benchmarks/java/Hashtable.java
import java.util.HashMap;

public class Hashtable {
    public static void main(String[] args) {
        HashMap<Integer, Integer> table = new HashMap<>();

        // Insert 10000 key-value pairs
        for (int i = 0; i < 10000; i++) {
            table.put(i, i * 2);
        }

        // Lookup 10000 values
        int total = 0;
        for (int i = 0; i < 10000; i++) {
            if (table.containsKey(i)) {
                int value = table.get(i);
                total += value;
            }
        }

        System.exit(0);
    }
}
